bet_log.json

```diff
@@ -0,0 +1,30 @@
+{
+    "current_bet_id": 4,
+    "bet_1": {
+        "bet_id": 1,
```

Comment:
any reason why the id is stored twice

Reply:
I thought I'd need it as an int more often, so it would be useful to have it stored in the bet definition. I also needed unique keys to add to the json without overwriting, not sure if there's another way to do that.